examples/flutter_gallery/lib/demo/calculator/home.dart

```diff
@@ -106,9 +106,7 @@ class _CalculatorState extends State<Calculator> {
   }
 
   void handleDelTap() {
-    setState(() {
-      popCalcExpression();
-    });
+    setState(popCalcExpression);
```

Comment:
Please don't do this to setState. We're trying to make setState look like a block statement, where it's "obvious" that the statements inside the block are executed immediately. This change hides what's going on, it makes it look like we're taking a reference to call later or something.